pkgs/development/python-modules/mkdocs-macros-plugin/default.nix

```diff
@@ -0,0 +1,76 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  pytestCheckHook,
+  hjson,
+  jinja2,
+  mkdocs,
+  packaging,
+  pathspec,
+  python-dateutil,
+  pyyaml,
+  termcolor,
+  super-collections,
+  mkdocs-test,
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-macros-plugin";
+  version = "1.3.9";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "fralau";
+    repo = "mkdocs-macros-plugin";
+    tag = "v${version}";
+    hash = "sha256-bL7oWWDoF+zH34XSwFY2H9op/97zO43HS+oO6lNFEr4=";
+  };
+
+  # we do not have bew and mkdocs-d2-plugin is also not packaged in nixpkgs,
+  # so we skip the assertion
+  postPatch = ''
+    substituteInPlace test/plugin_d2/test_t2.py \
+      --replace-fail 'brew' 'false'
```

Comment:
I would rather add `test/plugin_d2/test_t2.py` to `disabledTestPaths` (with a comment stating the reason)